src/main/features/response/views/macro/check-and-send/admission.njk

```diff
@@ -11,7 +11,7 @@
   }}
 {% endmacro %}
 
-{% macro notImmediatePayment(type, paymentDate, paymentPlan, explanation, Paths = {}, externalId) %}
+{% macro notImmediatePayment(type, paymentDate, paymentPlan, explanation, Paths = {}, externalId, statementOfMeansIsApplicable) %}
```

Comment:
I would prefer to pass undefined `explanation` if statement of means is not applicable.
Then in the macro we would have `Why you can’t pay now` section optional.

Reply:
I would prefer to keep the check the same for SoM features, its just a shame that the SoM task is in there too